local-dev/lagoon-logging.values.yaml

```diff
@@ -2,6 +2,9 @@ logsDispatcher:
   serviceMonitor:
     enabled: false
   replicaCount: 1
+  image:
+    repository: testlagoon/logs-dispatcher
+    tag: pr-3009
```

Comment:
I guess this should be removed before merge?

Reply:
ah yes - that's the merged PR for fluent plugin updates #3009